src/programs/CMakeLists.txt

```
@@ -64,20 +64,58 @@ foreach( program ectrans-benchmark )
 endforeach( program )
 
 if( HAVE_ETRANS )
-  foreach( prec sp dp )
-    if( HAVE_${prec} )
-      ecbuild_add_executable( TARGET  ectrans-lam-benchmark-cpu-${prec}
-                              SOURCES ectrans-lam-benchmark.F90
-                              LINKER_LANGUAGE Fortran
-                              LIBS
-                                fiat
-                                parkind_${prec}
-                                trans_${prec}
-                                etrans_${prec}
-                                OpenMP::OpenMP_Fortran
-                            )
-    endif()
-  endforeach()
+	foreach( program ectrans-lam-benchmark )
+
+	  if ( HAVE_CPU )
+		foreach( prec dp sp )
+		  if( HAVE_${prec} )
+			ecbuild_add_executable( TARGET  ${program}-cpu-${prec}
+									SOURCES ${program}.F90
+									LINKER_LANGUAGE Fortran
+									LIBS
+									  fiat
+									  parkind_${prec}
+									  trans_${prec}
+									  etrans_${prec}
```

```suggestion
									LIBS
									  fiat
									  parkind_${prec}
									  trans_${prec}
									  etrans_${prec}
									  $<${HAVE_OMP}:OpenMP::OpenMP_Fortran>
```
Hopefully this fixes the build time error due to missing `omp_wtime` symbol.